load-test/scripts/loadTest.js

```diff
@@ -18,14 +18,14 @@ import {
 // Change these options to increase the user goal or time to run the test.
 export const options = getTestOptions(
   /* User goal */ 1000,
-  /* High load time minutes */ 15
+  /* High load time minutes */ 4
 );
 
 // Change this to test different code
 const sourceToTest = helloWorld;
 // Set this to true to space out requests every REQUEST_TIME_MS milliseconds. Set to
 // false to send as many requests as possible.
-const SHOULD_SLEEP = true;
+const SHOULD_SLEEP = false;
```

Comment:
nit: should `SHOULD_SLEEP` be part of the `options` object just so all the configurable parameters are in one place?

Reply:
The [options object](https://k6.io/docs/getting-started/running-k6/#using-options) is used by k6 to determine specific runtime parameters and `SHOULD_SLEEP` is something we are using manually in our execution function (in `loadTest.js` we don't actually reference `options` anywhere, we just have to define it in the script). My thought with putting this here is that test-specific parameters are all configured here (the parameters to options, `sourceToTest` and `SHOULD_SLEEP`). I'll update `sourceToTest` to also be in all caps to imply it's a constant here, looking at this I realize both variables should have the same casing.